src/Upgrader/Communication/Command/UpgradeCommand.php

```diff
@@ -58,6 +83,19 @@ public function execute(InputInterface $input, OutputInterface $output): int
         return $upgraderResponseCollection->getExitCode();
     }
 
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     *
+     * @return \Upgrader\Business\Upgrader\Request\UpgraderRequest
+     */
+    protected function createRequest(InputInterface $input): UpgraderRequest
+    {
+        $strategyOption = $input->getOption(static::OPTION_STRATEGY) ?? UpgradeStrategyEnum::COMPOSER_UPDATE;
+        $strategyEnum = new UpgradeStrategyEnum($strategyOption);
+
+        return new UpgraderRequest($strategyEnum);
+    }
```

Comment:
could we use "create*"" method only in the Factory?
we might remove createRequest method and create DTO in the "execute" method directly, without dependency on the UpgradeStrategyEnum

Reply:
Deleted Enum